src/components/CohortBuilder/ParticipantsTableView/index.js

```diff
@@ -54,68 +54,91 @@ const ParticipantsTableView = ({
           );
         }
 
+        const dataTotalCount = data[0] ? data[0].total : 0;
```

Comment:
@jaouad-benassila I just read the ticket- it says to export "per page" rather than the entire dataset.